tracer/src/Datadog.Trace/AppSec/Waf/ReturnTypes.Managed/DiagnosticResultUtils.cs

```diff
@@ -7,6 +7,7 @@
 using System;
 using System.Collections.Generic;
 using Datadog.Trace.AppSec.Waf.NativeBindings;
+using Datadog.Trace.AppSec.WafEncoding;
```

Comment:
This is the only change of the file, is it required?

Reply:
not the only change but yes uneccesary, fixed in [6dd3d0e](https://github.com/DataDog/dd-trace-dotnet/pull/4884/commits/6dd3d0eed0ede7f95fda2eed4301f6cdaccb9138)